### ISSUE TITLE
Doesn't work with Python 3.9

### ISSUE THREAD

**Issue by redblade7** [gunthercox/ChatterBot#2081]
ChatterBot depends on an older version of spacy (2.1.9) which fails to install on Python 3.9.

**Comment by fccoelho** [comment]
It also fails on Python 3.8

**Comment by IacopoOrtis** [comment]
It also fails on Python 3.7

**Comment by redblade7** [comment]
> 
> 
> It also fails on Python 3.9 , 3.8 , 3.7
> `error: command '/usr/bin/clang' failed with exit code 1`
> 
> It **DOES WORK** on Python 3.6. I've read online it's a problem by "spacy" that is not yet ready.

spacy has since been updated, but ChatterBot forces an older version of spacy

**Comment by mkkeffeler** [comment]
yep, this is well documented on many issues, we need a maintainer to help get https://github.com/gunthercox/ChatterBot/pull/1963 merged in.

**Comment by sylvaline** [comment]
Please how to i get my bot to work? I use python 3.9.3 and my ChatBot is not working

**Comment by 01miaom** [comment]
sound terrible😭

**Comment by v1s1t0r999** [comment]
And it is terrible
